src/config.ts

```diff
@@ -13,6 +13,7 @@ export default {
     'madam',
     'maam',
     "ma'am",
+    "kid",
```

Comment:
is kid needed to be blocked?

Reply:
If yess because some members can use it to Bully young Dev's

Reply:
Ig we also have to add noob

Reply:
`noob` is a good one to add, I don't see people use it, but still worth adding 👍